meson_options.txt

```diff
@@ -1,3 +1,7 @@
+option('doc',
```

Comment:
let's make this `documentation`, immediately more obvious and most users will only ever have to type this once anyway.